### PR TITLE
*: remove pingcap/check dependency

### LINKED ISSUE #28328: migrate test-infra to testify for `session/session_test.go`
* [x] #34292
* [x] #34340
* [x] #34345

### PR DESCRIPTION
**Pull request by tisonkun**
Issue Number: close #28328 

```release-note
None
```

### CODE REVIEW

File: session/main_test.go [hunk 1]
```diff
@@ -39,6 +38,9 @@ import (
 
 var testDataMap = make(testdata.BookKeeper, 1)
 
+var _ = flag.Bool("with-tikv", false, "workaroundGoCheckFlags: with-tikv")
+var _ = flag.String("pd-addrs", "", "workaroundGoCheckFlags: pd-addrs")
```
Comment (tisonkun) [inline session/main_test.go:42]: We should remove these lines after removing corresponding tasks in PingCAP-QE/ci.

**Review by hawkingrei — APPROVED** [review]

File: session/main_test.go (hunk 1 above)
Comment (hawkingrei) [reply session/main_test.go:42]: OK, I will refactor the CI script recently.

**Review by tangenta — APPROVED** [review]

File: session/main_test.go (hunk 1 above)
Comment (tangenta) [reply session/main_test.go]: https://github.com/PingCAP-QE/ci/pull/975

**Review by bb7133 — APPROVED** [review]
LGTM

Great work! @tisonkun , thank you for everything you have done.

**Pull request merged**